packages/components/time-picker/__tests__/time-picker.spec.ts

```diff
@@ -613,7 +620,7 @@ describe('TimePicker(range)', () => {
     const input = wrapper.find('input')
     input.trigger('blur')
     input.trigger('focus')
-    await nextTick()
+    await sleep(200)
```

Comment:
Can we use `jest.runAllTimers()` here?

Reply:
Sorry for the wrong commit! I will revert back to nextTick!